### PR TITLE
Update dependencies.

### PR DESCRIPTION
**Pull request by DFurnes**
This pull request updates this library's Babel, ESLint & Webpack dependencies, and installs Prettier. This clears up some warnings we had from older sub-dependencies & brings this package in line with the rest of our applications.

### CODE REVIEW

**Comment by DFurnes** [comment]
The webhook seems to have gotten "unhooked", but here's this PR's [passing Wercker build](https://app.wercker.com/dosomething/forge/runs/build/5d07d30d87d3180008eba6ac).

File: package.json [hunk 1]
```diff
@@ -5,51 +5,78 @@
...
+      "pre-commit": "pretty-quick --staged"
+    }
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/DoSomething/forge.git"
   },
   "dependencies": {
-    "jquery": "^3.0.0",
+    "jquery": "^3.4.1",
```
Comment (weerd) [inline package.json:53]: forgot we're still using this! haha

Comment (DFurnes) [reply package.json:53]: Haha, I know! I'd love to remove this dependency, maybe during our cleanup sprint!

**Review by weerd — APPROVED** [review]
👍 🎨

**Pull request merged**